src/uwtools/api/chgres_cube.py

```diff
@@ -2,12 +2,13 @@
 API access to the ``uwtools`` ``chgres_cube`` driver.
 """
 
+from functools import partial
+
 from uwtools.drivers.chgres_cube import ChgresCube
-from uwtools.drivers.support import graph
+from uwtools.drivers.support import graph, tasks
 from uwtools.utils.api import make_execute as _make_execute
-from uwtools.utils.api import make_tasks as _make_tasks
 
 _driver = ChgresCube
 execute = _make_execute(_driver, with_cycle=True)
-tasks = _make_tasks(_driver)
+tasks = partial(tasks, _driver)
 __all__ = [_driver.__name__, "execute", "graph", "tasks"]
```

Comment:
This step is repeated in all internal drivers. `_make_tasks()` is made obsolete if we import `partial()` from `functools`.